test/immutable-store-state.js

```diff
@@ -0,0 +1,40 @@
+import Alt from '../dist/alt-with-runtime'
+import Immutable from 'immutable'
+import { assert } from 'chai'
+import sinon from 'sinon'
+
+const alt = new Alt()
+
+const actions = alt.generateActions('fire')
+
+const assign = require('object-assign')
+
+const store = alt.createStore({
+  displayName: 'ImmutableStore',
+
+  state: Immutable.Map({}),
```

Comment:
Should it be initialized as Immutable from the beginning ? What if user will want use other immutable data structures from Immutable.js?